script/build.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+# Build LightTable app and CLI and place in builds/.
+# Specify $VERSION to override default build version.
+# This script primarily installs dependencies and sets up
+# the app before calling build-app.sh to build it.
+
+# Check if lein is installed
+lein version >/dev/null 2>&1 || { echo >&2 "Please install leiningen before running this script."; exit 1; }
+if [ "$(echo `lein version` | grep 'Leiningen \(1.\|2.0\)')" ]; then
+  echo "lein version must be 2.1 or above. Do a lein upgrade first"; exit 1;
+fi
+
+# Check if npm is installed
+npm --version >/dev/null 2>&1 || { echo >&2 "Please install npm before running this script."; exit 1; }
+
+# Ensure we start in project root
+cd "$(dirname "${BASH_SOURCE[0]}")"; cd ..
+
+# Ensure we have current version of atom-shell
+pushd shell
+  npm install -g grunt-cli
```

Comment:
I just got this error. Grunt manual says it may require sudo access to install `grunt-cli`. 

```
~/projects/lighttable/shell ~/projects/lighttable
npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "-g" "grunt-cli"
npm ERR! node v0.10.33
npm ERR! npm  v2.1.8
npm ERR! path /usr/local/bin/grunt
npm ERR! code EACCES
npm ERR! errno 3

npm ERR! Error: EACCES, unlink '/usr/local/bin/grunt'
npm ERR!  { [Error: EACCES, unlink '/usr/local/bin/grunt'] errno: 3, code: 'EACCES', path: '/usr/local/bin/grunt' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.
npm ERR! error rolling back Error: EACCES, unlink '/usr/local/bin/grunt'
npm ERR! error rolling back  { [Error: EACCES, unlink '/usr/local/bin/grunt'] errno: 3, code: 'EACCES', path: '/usr/local/bin/grunt' }

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/brabadu/projects/lighttable/shell/npm-debug.log
```